package.json

```diff
@@ -15,6 +15,7 @@
     "bootswatch": "^3.3.7",
     "jwt-decode": "^2.2.0",
     "moment": "^2.18.1",
+    "prop-types": "^15.6.0",
```

Comment:
PropTypes are needed to access context from React Bootstrap (or any context, btw).